smart-contract/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dcc"
-version = "0.5.1"
+version = "0.5.3"
```

Comment:
installed the wrong binary on version 0.5.2 on test so just rev'ed to 0.5.3 for testnet/mainnet consistency